packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Improved output formatting for `zowe zos-tso start app` and `zowe zos-tso send app`[#2347](https://github.com/zowe/zowe-cli/pull/2347)
```

Comment:
We should add more detail about how the formatting was improved: "Improved the output formatting for `aaaa` and `bbbb` to parse out and display only relevant/necessary/appropriate/important data."